src/components/content/medication-dispense/helpers/dispense-columns.tsx

```diff
@@ -0,0 +1,25 @@
+import { TableColumn } from "@/components/core/table/table-helpers";
+import { MedicationDispenseModel } from "@/fhir/models";
+
+export const patientMedicationDispenseColumns = () => {
+  const dispenseColumns: TableColumn<MedicationDispenseModel>[] = [
+    {
+      title: "TimeStamp",
+      render: (m) => <div>{m.alertTimeStamp}</div>,
+    },
+    {
+      title: "Medication",
+      render: (m) => <div>{m.medicationDisplayName}</div>,
+    },
+    {
+      title: "Performer Details",
+      render: (m) => <div>{m.performerDetails.name}</div>,
+    },
+    {
+      title: "Last Known Fill",
+      render: (m) => <div>{m.whenHandedOver ? m.whenHandedOver : m.whenPrepared}</div>,
+    },
+  ];
```

Comment:
Were these in divs for styling? If not we should just stick to
```suggestion
  const dispenseColumns: TableColumn<MedicationDispenseModel>[] = [
    {
      title: "TimeStamp",
      render: (m) => m.alertTimeStamp,
    },
    {
      title: "Medication",
      render: (m) => m.medicationDisplayName,
    },
    {
      title: "Performer Details",
      render: (m) => m.performerDetails.name,
    },
    {
      title: "Last Known Fill",
      render: (m) => m.whenHandedOver ? m.whenHandedOver : m.whenPrepared,
    },
  ];
```